user/store.go

```diff
@@ -104,9 +106,11 @@ func NewStore(ctx context.Context, db *sql.DB) (*Store, error) {
 
 		findOne: p.P(`
 			SELECT
-				id, name, email, avatar_url, role, alert_status_log_contact_method_id
-			FROM users
-			WHERE id = $1
+				u.id, u.name, u.email, u.avatar_url, u.role, u.alert_status_log_contact_method_id, fav is distinct from null
+			FROM users u
+			LEFT JOIN user_favorites fav ON
+				fav.tgt_user_id = u.id AND fav.user_id = $2
```

Comment:
`userID` will need to be passed when the `findOne` statement is queried, but might need separate returns where `findOne` and `findOneForUpdate` are queried since `findOneForUpdate` doesn't use `userID`.

Reply:
Addressed, fixed bug by passing additional `userID` into `findOne`. Previous behavior observed was a failure to render _all_ user profiles with individual avatars.